ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -1,137 +1,133 @@
+description: Learn how to deploy Calico route reflectors to improve the performance of Kubernetes networking.
 keywords: cluster, node, label, certificate, SAN
 ---
 
 >{% include enterprise_label_shortform.md %}
 
-UCP uses Calico as the default Kubernetes networking solution. Calico is
-configured to create a BGP mesh between all nodes in the cluster.
+Docker Universal Control Plane (UCP) uses [Calico](https://docs.projectcalico.org/v3.11/introduction/) as the default Kubernetes networking solution. Calico is configured to create a Border Gateway Protocol (BGP) mesh between all nodes in the cluster.
 
 As you add more nodes to the cluster, networking performance starts decreasing.
 If your cluster has more than 100 nodes, you should reconfigure Calico to use
-Route Reflectors instead of a node-to-node mesh.
+route reflectors instead of a node-to-node mesh.
 
-This article guides you in deploying Calico Route Reflectors in a UCP cluster.
+Route reflectors are useful in large scale deployments to reduce the number of BGP connections needed for correct and complete route propagation.
+
+This topic guides you in deploying Calico route reflectors in a UCP cluster.
 UCP running on Microsoft Azure uses Azure SDN instead of Calico for
-multi-host networking.
-If your UCP deployment is running on Azure, you don't need to configure it this
-way.
+multi-host networking. If your UCP deployment is running on Azure, you don't need to configure it this way. See [Install UCP on Azure](https://docs.docker.com/ee/ucp/admin/install/cloudproviders/install-on-azure/) for more information.
 
 ## Before you begin
 
-For production-grade systems, you should deploy at least two Route Reflectors,
+For production-grade systems, you should deploy at least two route reflectors,
 each running on a dedicated node. These nodes should not be running any other
 workloads.
 
-If Route Reflectors are running on a same node as other workloads, swarm ingress
+If route reflectors are running on the same node as other workloads, swarm ingress
 and NodePorts might not work in these workloads.
 
 ## Choose dedicated nodes
 
-Start by tainting the nodes, so that no other workload runs there. [Configure
-your CLI with a UCP client bundle](/ee/ucp/user-access/cli/), and for each dedicated node, run:
+1. Start by tainting the nodes so that no other workload runs there. See [Restrict services to worker nodes](https://docs.docker.com/ee/ucp/admin/configure/restrict-services-to-worker-nodes/) for more information on taints.
+2. Configure your CLI with a [UCP client bundle](/ee/ucp/user-access/cli/).
+3. For each dedicated node, run:
 
-```
-kubectl taint node <node-name> \
-  com.docker.ucp.kubernetes.calico/route-reflector=true:NoSchedule
-```
+    ```
+    kubectl taint node <node-name> \
+    com.docker.ucp.kubernetes.calico/route-reflector=true:NoSchedule
+    ```
+4. Add labels to those nodes to target them when deploying the route reflectors. For each dedicated node, run:
```

Comment:
Edit step 4 (now step 2) to read:

" Add labels to each of the dedicated nodes."